recordings/basic-setup/run.sh

```diff
@@ -0,0 +1,309 @@
+run <<< "# Note that this part depend on your infra, disk names or other properties."
+run <<< "# Be sure to double check the NodeConfig and adjust it, if needs be."
+run <<'EOF'
+yq -e eval --colors '.' ./examples/gke/nodeconfig-alpha.yaml | head -n 25
+EOF
+sleep-extra 10
+run <<'EOF'
+kubectl apply --server-side -f=./examples/gke/nodeconfig-alpha.yaml
+EOF
+
+run <<< "# Wait for the NodeConfig to apply changes to the Kubernetes nodes."
+run <<'EOF'
+kubectl wait --for='condition=Reconciled' --timeout=10m nodeconfigs.scylla.scylladb.com/cluster
+EOF
+# TODO: Extend waiting for the NodeConfig when we fix https://github.com/scylladb/scylla-operator/issues/1557
+
+run <<< "# Install ScyllaDB local CSI driver that will provision storage for PersistentVolumeClaims."
+run <<< "# The ScyllaDB local CSI driver is configured to use folder /mnt/persistent-volumes that was created by the NodeConfig."
+run <<'EOF'
+kubectl -n local-csi-driver apply --server-side -f=https://raw.githubusercontent.com/scylladb/k8s-local-volume-provisioner/v0.2.0-rc.0/{deploy/kubernetes/local-csi-driver/{00_namespace,10_csidriver,10_driver_serviceaccount,10_provisioner_clusterrole,20_provisioner_clusterrolebinding},example/storageclass_xfs}.yaml
+kubectl create -f=https://raw.githubusercontent.com/scylladb/k8s-local-volume-provisioner/v0.2.0-rc.0/deploy/kubernetes/local-csi-driver/50_daemonset.yaml --dry-run=client -o yaml | yq -e '.spec.template.spec.nodeSelector += {"scylla.scylladb.com/node-type": "scylla"}' | kubectl -n=local-csi-driver apply --server-side -f -
+kubectl -n=scylla-operator rollout status --timeout=10m deployment.apps/{scylla-operator,webhook-server}
+EOF
+run <<'EOF'
+kubectl -n=local-csi-driver rollout status --timeout=10m daemonset.apps/local-csi-driver
+EOF
+
+run <<< "# Because all the following manifests use default storage class, we'll set scylladb-local-xfs as the default one, for simplicity."
+run <<< "# As there can be only one default storage class, make sure to unset the previous default one, if it existed in your setup."
+run <<< "# This can be also controlled per ScyllaCluster."
+run <<'EOF'
+kubectl patch storageclass scylladb-local-xfs -p '{"metadata": {"annotations":{"storageclass.kubernetes.io/is-default-class":"true"}}}'
+EOF
+sleep-extra 5
+
+run <<< "# Optionally, deploy Scylla Manager"
+run <<< "# Scylla Manager is global and shared by every ScyllaCluster"
+run <<'EOF'
+kubectl -n=scylla-manager apply --server-side -f=./deploy/manager/dev/
+EOF
+run <<'EOF'
+kubectl -n=scylla-manager rollout status --timeout=10m deployment.apps/scylla-manager{,-controller}
+EOF
+
+run <<< "# We have now successfully deployed the infrastructure and can start creating ScyllaDB clusters."
+
+run <<< "# Create demo namespace"
+run <<'EOF'
+kubectl create namespace demo --dry-run=client -o=yaml | kubectl apply --server-side -f=-
+EOF
+
+run <<< "# Optionally, create a config for ScyllaDB cluster"
```

Comment:
```suggestion
run <<< "# Optionally, create a config for ScyllaDB clusters."
```
nit, for consistency

Reply:
there is just 1 ScyllaDB cluster